features/buyer/buyer_dashboard.feature

```diff
@@ -0,0 +1,31 @@
+@skip-staging @skip-production
+@buyer-dashboard
+Feature: Buyer Dashboard
+
+Scenario: Users should see new Dashboard
+  Given I am logged in as a buyer user
+  And I am on the /buyers page
+  Then I see 'Cloud hosting, software and support' text on the page
+  And I see 'Digital outcomes, specialists and user research' text on the page
+
+Scenario: Users should see link when there are searches available to view
+  Given I am logged in as a buyer user
+  And I have a saved search
+  And I am on the /buyers page
+  Then I see the 'View your saved searches' link
+
+Scenario: Users should see message when no searches are created
+  Given I am logged in as a buyer user
+  And I am on the /buyers page
+  Then I see 'You don't have any saved searches' text on the page
+
+Scenario: Users should see link when there are requirements available to view
```

Comment:
Yeah, so per @samuelhwilliams's earlier comment, I think this test is entirely covered by the existing test in `requirements.feature` called "View requirement in a dashboard". This doesn't test anything that doesn't test already, as far as I can see.

So let's remove the duplicative ones like this, and see what we're left with. Testing the case where there's no requirements / no G-Cloud projects is okay, I guess.

And we need to make sure we've got a G-Cloud equivalent of that DOS test, i.e. one where you actually click through all the way from the home page to right down to the task list.

And somewhere we want a test that takes a project right through all the stages, i.e. End Search and Download. That could be here too, unless it's already somewhere else and I've missed it? cheers